aws_lambda_powertools/logging/lambda_context.py

```diff
@@ -31,7 +34,7 @@ def __init__(
         self.function_request_id = function_request_id
 
 
-def build_lambda_context_model(context: object) -> LambdaContextModel:
+def build_lambda_context_model(context: Any) -> LambdaContextModel:
```

Comment:
`context: Any`

I wonder if we could use this implementation as a type to have a more realistic representation of what we expect as input: https://github.com/aws/aws-lambda-python-runtime-interface-client/blob/main/awslambdaric/lambda_context.py

Reply:
Future idea? Maybe part of the testing libs as well?